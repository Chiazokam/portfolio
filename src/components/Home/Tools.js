import React from 'react';
import Container from '@material-ui/core/Container';

const Tools = () => {
    return (
        <section className='tools-section'>
            <Container fixed>
                <div className='section-title'>What I Have Worked With...</div>
            </Container>
        </section>
    )
}

export default Tools;
