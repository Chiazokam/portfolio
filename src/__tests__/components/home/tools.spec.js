import React from 'react';
import { shallow } from 'enzyme';
import Tools from '../../../components/Home/Tools';

describe('Tools component', () => {
    it('renders successfully', () => {
    const wrapper = shallow(<Tools />);
    expect(wrapper).toMatchSnapshot();
    })
});
