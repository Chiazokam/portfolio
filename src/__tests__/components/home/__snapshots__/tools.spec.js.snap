// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tools component renders successfully 1`] = `ShallowWrapper {}`;
